### PR TITLE
feat(cat-voices): registration components

### PR DESCRIPTION
**Pull request by damian-molinski**
# Description

Extracting more common widgets into separate widgets for better clarity.

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: catalyst_voices/lib/pages/registration/registration_stage_navigation.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:flutter/material.dart';
+
+class BackNextNavigation extends StatelessWidget {
```
Comment (dtscalac) [inline catalyst_voices/lib/pages/registration/registration_stage_navigation.dart:6]: RegistrationBackNextNavigation? you called the file name after registration

Comment (damian-molinski) [reply catalyst_voices/lib/pages/registration/registration_stage_navigation.dart]: Well, yes. I can see that.

Comment (damian-molinski) [reply catalyst_voices/lib/pages/registration/registration_stage_navigation.dart]: fixed

**Review by dtscalac — APPROVED** [review]
lgtm

**Pull request merged**